src/core/Table/BasicTable/BasicTable.tsx

```diff
@@ -0,0 +1,60 @@
+import {
+  Table,
+  TableBody,
+  TableContainer,
+  TableHead,
+  TableProps,
+  TablePagination,
+  Paper,
+} from '@material-ui/core';
+import React from 'react';
+import { useStyles } from './style';
+
+interface PaginationData {
+  pageNo: number;
+  rowsPerPage: number;
+}
+
+interface BasicTableProps extends TableProps {
```

Comment:
you can get the `TableProps` from another file as **base.ts**, have a look as done in other components.